sources/packages/api/src/route-controllers/utils/address-utils.ts

```diff
@@ -16,6 +16,7 @@ export const COUNTRY_CANADA = "canada";
 export function transformAddressDetailsForAddressBlockForm(
   addressDetails: AddressInfo,
 ): AddressDetailsAPIOutDTO {
+  const selectedCountry = addressDetails.selectedCountry.toLowerCase();
```

Comment:
may I know the reason to use `toLowerCase();` ?

Reply:
The country name is being displayed in lower case in the UI. Added a lower case comparison to method transformAddressDetailsForAddressBlockForm to allow us to change the country dropdown code in form.io later to have Canada in the proper case.

Reply:
mm, got it... so how about changing the dropdown value itself to `Canada`, so that we can avoid these case transformations? just a thought.

Reply:
As we talked about, the form.io definitions will be changed in upcoming PRs and the idea to have the lowerCase is to avoid future issues if someone inadvertently change it.